src/NuGet.Clients/NuGet.VisualStudio.OnlineEnvironment.Client/NuGetClientPackage.vsct

```diff
@@ -71,6 +72,7 @@
         <CommandFlag>AllowParams</CommandFlag>
         <CommandFlag>DefaultInvisible</CommandFlag>
         <CommandFlag>DynamicVisibility</CommandFlag>
+        <CommandFlag>AllowClientRole</CommandFlag>
```

Comment:
Do you also need to add the flag in https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Clients/NuGet.Tools/NuGetTools.vsct ?

Reply:
No. 
NuGetClientPackage is the only ones with menu items compatible in VSO right now.